tests/Tests/500__jupyterhub/pvc-change_verification.robot

```diff
@@ -0,0 +1,124 @@
+*** Settings ***
+Documentation    PVC_CHANGE_VERIFICATION
```

Comment:
I added another Cluster Settings test in  tests/Tests/400__ods_dashboard/410__ods_dashboard_settings.robot

What do you think about adding it there or in a new file  tests/Tests/400__ods_dashboard/411__ods_dashboard_settings_pvc_size.robot ?

Reply:
I have made moved to dashboard folder and closed this PR since there was some complications